docs/developers/tooling/data-indexers/dipdup/overview.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+sidebar_position: 1
+image: /img/socialCards/dipdup.jpg
+---
+
+[DipDup](https://dipdup.io/) is a Python framework for building smart contract indexers. It helps developers focus on business logic instead of writing a boilerplate to store and serve data. DipDup-based indexers are selective, which means only required data is requested. This approach allows to achieve faster indexing times and decreased load on underlying APIs.
+
+This page will guide you through the steps to get your first DipDup indexer up and running in a few minutes without getting too deep into the details.
```

Comment:
It feels like lines 9-11 belong more in the quickstart file as an intro than they do here?